libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -53,6 +52,8 @@ class AC_PosControl
     void set_dt(float dt) { _dt = dt; }
     float get_dt() const { return _dt; }
 
+    void update_estimates(bool high_vibes = false);
```

Comment:
we should probably add a comment above this declaration, we might need the same thing in the .cpp file as well

Reply:
Done.